src/features/broker-protection/actions/extract.js

```diff
@@ -191,7 +194,7 @@ export function aggregateFields (profile) {
         addresses,
         phoneNumbers,
         relatives: profile.relativesList,
-        profileUrl: profile.profileUrl
+        ...profile.profileUrl
```

Comment:
Identifier here seems a little ambiguous as a field name. I don't really understand the code if that matters but it could be confusing further down the line.

Reply:
Thanks @jonathanKingston, I think it's ok in the context of the object returned (which is a profile-specific object). Unless @Bunn disagrees, I think it's ok to keep it as `identifier` for now.

Reply:
yeah, it's an identifier of the object in the context. LGTM